src/config/ConfigOptions.ts

```diff
@@ -65,11 +65,19 @@ export interface ConfigOptions {
   };
 
   /**
-   * Allow to join a group calls without audio and video.
-   * TEMPORARY: Is a feature that's not proved and experimental
+   * TEMPORARY experimental features.
    */
   features?: {
-    feature_group_calls_without_video_and_audio: boolean;
+    /**
+     * Allow to join group calls without audio and video.
+     */
+    feature_group_calls_without_video_and_audio?: boolean;
+    /**
+     * Send device-specific call session membership state events
+     * instead of the legacy user-specific call membership state events,
+     * but not in rooms that contain active calls with legacy state events.
```

Comment:
```suggestion
     * Send device-specific call session membership state events
     * instead of the legacy user-specific call membership state events.
     * This setting has not effect when the user joins an active call with legacy state events.
     * For compatibility Element Call will always join running legacy calls with legacy state events.```